### ISSUE TITLE
Changing LAT/LON on existing install

### ISSUE THREAD

**Issue by dlasher** [tsunghanliu/adsb-box.snap#53]
Moved, needed to update LAT/LON - these commands:
`$ sudo snap set adsb-box receiver.latitude=[LATITUDE] receiver.longitude=[LONGITUDE] receiver.altitude=[ALTITUDE]`

don't change the existing values.. changed while running, stopped, changed, started, rebooted, old values stay in place. Had to edit all the files by hand to get it to change.

Did I miss something?

**Comment by tsunghanliu** [comment]
Hi,

Did you run `sudo snap restart adsb-box` after change the LAT/LON values?
What files did you change manually? I can check if something is not generated dynamically.

**Comment by dlasher** [comment]
Yes, I tried restarting it, and all the services started with the old LAT/LON.

Hand edited the following files, then restarted:

* dump1090-fa.conf
* location.env
* location
* fr24feed.ini
* 10-debconf.ini
* rbfeeder.ini

**Comment by tsunghanliu** [comment]
Thanks for the information. I didn't notice that feeders recorded the location in files separately.
However, my `fr24feed.ini does have the location stored. Could you please share the format with me?

**Comment by dlasher** [comment]
Apologies, you're right, mine doesn't either.  I looked back through my bash history, and included every file I edited, and that one does NOT have location data, so we can ignore it.
